services/src/main/java/io/grpc/services/CallMetricRecorder.java

```diff
@@ -55,9 +55,9 @@ public static final class CallMetricReport {
     /**
      * Create a report for all backend metrics.
      */
-    CallMetricReport(double cpuUtilization, double memoryUtilization,
-                                   Map<String, Double> requestCostMetrics,
-                                   Map<String, Double> utilizationMetrics) {
+    public CallMetricReport(double cpuUtilization, double memoryUtilization,
```

Comment:
Why was this made public? I don't think this API is in a position to become public.